BangazonAPI/BangazonAPI/Models/ComputerEmployee.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class ComputerEmployee
+    {
+        public int Id { get; set; }
+
+        public int EmployeeId { get; set; }
+
+        public int ComputerId { get; set; }
+
```

Comment:
We don't need this model- we'll never be exposing the join table directly to the users, just using it as a bridge in our SQL queries. You can delete it!